libs/core/src/lib/mega-menu/mega-menu.component.ts

```diff
@@ -27,4 +32,12 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MegaMenuComponent {}
+export class MegaMenuComponent {
+    dir$: BehaviorSubject<string> = new BehaviorSubject(rtl);
+
+    constructor(@Optional() private rtlService: RtlService) {
+        if (rtlService) {
+            rtlService.rtl.subscribe(isRtl => this.dir$.next(isRtl ? rtl : ltr));
```

Comment:
Why use the ltr/rtl consts here when we can just use the boolean from the service?